concourse/tasks/update-task-definition.yml

```diff
@@ -0,0 +1,38 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/terraform
+    tag: terraform-0.13.3
```

Comment:
It's gonna be fun finding a nice way to keep these versions in sync 😅 (not something to worry about right now though).